src/App.jsx

```diff
@@ -14,45 +17,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_ALPHABETICALLY = 'Sort alphabetically';
+const SORT_LENGTH = 'Sort by length';
+const SORT_REVERSE = 'Reverse';
+
+function getPreparedGoods(goods, { sortField }) {
+  const preparedGoods = [...goods];
+
+  if (sortField.length > 0) {
+    sortField.forEach((elem) => {
+      switch (elem) {
+        case SORT_ALPHABETICALLY:
+
+          return preparedGoods
+            .sort((good1, good2) => good1.localeCompare(good2));
+        case SORT_LENGTH:
+
+          return preparedGoods
+            .sort((good1, good2) => good1.length - good2.length);
+        case SORT_REVERSE:
+
+          return preparedGoods.reverse();
+        default:
+          return preparedGoods;
+      }
+    });
+  }
 
-    <ul>
-      <li data-cy="Good">Dumplings</li>
-      <li data-cy="Good">Carrot</li>
-      <li data-cy="Good">Eggs</li>
-      <li data-cy="Good">Ice cream</li>
-      <li data-cy="Good">Apple</li>
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState([]);
```

Comment:
You can use string as a default value and create another boolean state for reverse value